semantic-ast/semantic-ast.cabal

```diff
@@ -0,0 +1,44 @@
+cabal-version:       2.4
+-- Initial package description 'semantic-ast.cabal' generated by 'cabal
+-- init'.  For further documentation, see
+-- http://haskell.org/cabal/users-guide/
+
+name:                semantic-ast
+version:             0.1.0.0
+-- synopsis:
+-- description:
+-- bug-reports:
+license:             MIT
+license-file:        LICENSE
+author:              The Semantic Authors
+maintainer:          opensource+semantic@github.com
+-- copyright:
+category:            Language
+extra-source-files:  CHANGELOG.md
+
+library
+  exposed-modules:
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base ^>=4.12.0.0
+                     , tree-sitter ^>= 0.5
+                     , semantic-source ^>= 0.0
+                     , tree-sitter-python
+                     , bytestring
+                     , optparse-applicative ^>= 0.14.3.0
+  hs-source-dirs:      src
+  default-language:    Haskell2010
+
+executable semantic-ast
+  main-is:             Main.hs
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base ^>=4.12.0.0
+                     , semantic-ast
+                     , tree-sitter ^>= 0.5
+                     , semantic-source ^>= 0.0
+                     , tree-sitter-python
+                     , bytestring
+                     , optparse-applicative ^>= 0.14.3.0
```

Comment:
Technically, you don’t need to provide the bounds here, since you’ve specified them in the library component.